### PR TITLE
sends token with request, for auth purposes

### PR DESCRIPTION
**Pull request by c-melchor**
- This PR reconfigures the code that sends the token
- Previous code with {authState} from props was removed because it was returning undefined

### CODE REVIEW

File: src/components/pages/CaseTable/CaseTable.js [hunk 1]
```diff
@@ -148,7 +148,6 @@ export default function CaseTable(props) {
     axios
       .get(`${process.env.REACT_APP_API_URI}/case/${id}`)
       .then(res => {
-        console.log(res.data);
```
Comment (JusticeMatthew) [inline src/components/pages/CaseTable/CaseTable.js:151]: Nice catch on this log, I love clean code

**Review by JusticeMatthew — APPROVED** [review]
A good fix for the auth, very important!

**Review by ruizaj13 — APPROVED** [review]
Code fixes holes in the applications security which is super important, good work

**Review by cmirza — APPROVED** [review]
Changes all look understandable and necessary. No fluff. Nice!

**Review by crystal-csete — APPROVED** [review]
Glad to see you got those endpoints protected.

**Review by ndossett — APPROVED** [review]
Good job getting this auth fixed- verification is important with this kind of data!

**Pull request merged**